qa/integration-tests/src/test/java/io/camunda/it/client/DecisionInstanceQueryTest.java

```diff
@@ -341,7 +341,7 @@ private static EvaluateDecisionResponse evaluateDecision(
   private static void waitForDecisionsToBeEvaluated(
       final CamundaClient camundaClient, final int expectedCount) {
     Awaitility.await("should deploy decision definitions and import in Operate")
-        .atMost(Duration.ofSeconds(15))
+        .atMost(Duration.ofSeconds(120))
```

Comment:
TBH I'm not sure whether I like that we increase everywhere the timeouts 🤔 I'm wondering whether we could speed up the index creation somehow. 

I tried to parallelize it here https://github.com/camunda/camunda/pull/28433 let's see what the results are

Reply:
@ChrisKujawa do you not like the fact that we introduce wait time in general or that OS doesn't create indexes on time?

If concern is in general, we can make delay a CI-based variable. AWS OS CI tests have intention to run weekly once, hence I don't see a reason to optimize that: after all this `await` will hold 120 seconds in worst case but default polling interval is 100ms.

If concern is about AWS OS, your proposed solution might work.